client/app/ip/ip/firewall/game/ip-ip-firewall-game.constants.js

```diff
@@ -0,0 +1,8 @@
+export const IP_MITIGIATION_RULE_PROTOCOL_GAMES = {
+  gtaSanAndreasMultiplayerMod: 'GTA San Andreas Multiplayer Mode',
```

Comment:
I'm not sure to understand why we have to keep theses constants from the Front side.

Is there any reason?

Reply:
`mod => Mod`e and `hl2life => half life/source`. We cannot get the required text from `_.startCase`. So for some cases we are using this constant.